spring-cloud-gcp-logging/src/test/resources/logback-test.xml

```diff
@@ -29,10 +29,29 @@
 		</encoder>
 	</appender>
 
+	<appender name="CONSOLE_JSON_SERVICE_CTX_MISSING" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
Is the name right? the context configuration seems to be present here.

Reply:
It references an appender that has a missing `loggingEventEnhancer` - open to making it more clear, but it looks like Logback swallows a bunch of configuration exceptions (see test).